modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/openapi/v1_0/ObjectEntryOpenAPIResourceImpl.java

```diff
@@ -20,6 +20,7 @@
 import com.liferay.object.field.setting.util.ObjectFieldSettingUtil;
 import com.liferay.object.model.ObjectDefinition;
 import com.liferay.object.model.ObjectField;
+import com.liferay.object.model.ObjectRelationship;
```

Comment:
I miss certain tests for this functionality

Reply:
Should I create a new test class for integration tests related to this? Just to avoid coupling the new tests in classes that do not correspond

Reply:
As we talked, there is a class for the OpenAPI tests of the objects functionality in objects-rest-test module